### DISCUSSION TITLE
内容处于加密状态怎么解决

### DISCUSSION THREAD

**Discussion by QZHlinger** [pjy612/SteamManifestCache#19] [Question / Answer]
用SAl入库了只狼，缺少DLC的密匙导致现在卡下载了，关闭DLC也还是下不了咋办😭

**Comment by pjy612** [comment]
只狼很长时间没更新了，所以极大概率是密钥因为开着steam 导致被还原。
保险起见，请关闭Steam再入库。

**Reply by QZHlinger** [reply]
那现在想玩只狼怎么办，一下载就提示内容加密，关闭steam入库也还是一样: (

**Reply by pjy612** [reply]
？你确定用的 tools 1.8 beta 5，然后 密钥写入正常？
正常情况是能入的，别的不好说啥原因。
